packages/mediapipe-core/lib/src/task_options.dart

```diff
@@ -0,0 +1,158 @@
+///
+/// See also:
+///  * [MediaPipe's BaseOptions documentation](https://developers.google.com/mediapipe/api/solutions/java/com/google/mediapipe/tasks/core/BaseOptions)
+///  * [ClassifierOptions], which is often used in conjunction to specify a
+///    classifier's desired behavior.
+class BaseOptions extends Equatable {
+  /// Generative constructor that creates a [BaseOptions] instance.
+  const BaseOptions({this.modelAssetBuffer, this.modelAssetPath})
+      : assert(
+          !(modelAssetBuffer == null && modelAssetPath == null),
+          'You must supply either `modelAssetBuffer` or `modelAssetPath`',
+        ),
+        assert(
+          !(modelAssetBuffer != null && modelAssetPath != null),
+          'You must only supply one of `modelAssetBuffer` and `modelAssetPath`',
+        );
+
+  /// The model asset file contents as bytes;
+  final Uint8List? modelAssetBuffer;
+
+  /// Path to the model asset file.
+  final String? modelAssetPath;
+
+  /// Converts this pure-Dart representation into C-memory suitable for the
+  /// MediaPipe SDK to instantiate various classifiers.
+  Pointer<bindings.BaseOptions> toStruct() {
+    final struct = calloc<bindings.BaseOptions>();
+
+    if (modelAssetPath != null) {
+      struct.ref.model_asset_path = prepareString(modelAssetPath!);
+    }
+    if (modelAssetBuffer != null) {
+      struct.ref.model_asset_buffer = prepareUint8List(modelAssetBuffer!);
+    }
+    return struct;
+  }
+
+  @override
+  List<Object?> get props => [modelAssetBuffer, modelAssetPath];
+}
+
+/// Dart representation of MediaPipe's "ClassifierOptions" concept.
+///
+/// Classifier options shared across MediaPipe classification tasks.
+///
+/// See also:
+///  * [MediaPipe's ClassifierOptions documentation](https://developers.google.com/mediapipe/api/solutions/java/com/google/mediapipe/tasks/components/processors/ClassifierOptions)
+///  * [BaseOptions], which is often used in conjunction to specify a
+///    classifier's desired behavior.
+class ClassifierOptions extends Equatable {
+  /// Generative constructor that creates a [ClassifierOptions] instance.
+  const ClassifierOptions({
+    this.displayNamesLocale,
+    this.maxResults,
+    this.scoreThreshold,
+    this.categoryAllowlist,
+    this.categoryDenylist,
+  });
+
+  /// The locale to use for display names specified through the TFLite Model
+  /// Metadata.
+  final String? displayNamesLocale;
+
+  /// The maximum number of top-scored classification results to return.
+  final int? maxResults;
+
+  /// If set, establishes a minimum `score` and leads to the rejection of any
+  /// categories with lower `score` values.
+  final double? scoreThreshold;
+
+  /// Allowlist of category names.
+  ///
+  /// If non-empty, classification results whose category name is not in
+  /// this set will be discarded. Duplicate or unknown category names
+  /// are ignored. Mutually exclusive with `categoryDenylist`.
+  ///
+  /// See also:
+  ///  * [Category.categoryName]
```

Comment:
Same here, why would they want to see this too? We usually add a brief blurb of what or why the recommendation is listed